// uuid/dist/index.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
Object.defineProperty(exports, "MAX", {
  enumerable: true,
  get: function () {
    return _max.default;
  }
});
Object.defineProperty(exports, "NIL", {
  enumerable: true,
  get: function () {
    return _nil.default;
  }
});
Object.defineProperty(exports, "parse", {
  enumerable: true,
  get: function () {
    return _parse.default;
  }
});
Object.defineProperty(exports, "stringify", {
  enumerable: true,
  get: function () {
    return _stringify.default;
  }
});
Object.defineProperty(exports, "v1", {
  enumerable: true,
  get: function () {
    return _v.default;
  }
});
Object.defineProperty(exports, "v1ToV6", {
  enumerable: true,
  get: function () {
    return _v1ToV.default;
  }
});
Object.defineProperty(exports, "v3", {
  enumerable: true,
  get: function () {
    return _v2.default;
  }
});
Object.defineProperty(exports, "v4", {
  enumerable: true,
  get: function () {
    return _v3.default;
  }
});
Object.defineProperty(exports, "v5", {
  enumerable: true,
  get: function () {
    return _v4.default;
  }
});
Object.defineProperty(exports, "v6", {
  enumerable: true,
  get: function () {
    return _v5.default;
  }
});
Object.defineProperty(exports, "v6ToV1", {
  enumerable: true,
  get: function () {
    return _v6ToV.default;
  }
});
Object.defineProperty(exports, "v7", {
  enumerable: true,
  get: function () {
    return _v6.default;
  }
});
Object.defineProperty(exports, "validate", {
  enumerable: true,
  get: function () {
    return _validate.default;
  }
});
Object.defineProperty(exports, "version", {
  enumerable: true,
  get: function () {
    return _version.default;
  }
});
var _max = _interopRequireDefault(require("uuid/dist/max.js"));
var _nil = _interopRequireDefault(require("uuid/dist/nil.js"));
var _parse = _interopRequireDefault(require("uuid/dist/parse.js"));
var _stringify = _interopRequireDefault(require("uuid/dist/stringify.js"));
var _v = _interopRequireDefault(require("uuid/dist/v1.js"));
var _v1ToV = _interopRequireDefault(require("uuid/dist/v1ToV6.js"));
var _v2 = _interopRequireDefault(require("uuid/dist/v3.js"));
var _v3 = _interopRequireDefault(require("uuid/dist/v4.js"));
var _v4 = _interopRequireDefault(require("uuid/dist/v5.js"));
var _v5 = _interopRequireDefault(require("uuid/dist/v6.js"));
var _v6ToV = _interopRequireDefault(require("uuid/dist/v6ToV1.js"));
var _v6 = _interopRequireDefault(require("uuid/dist/v7.js"));
var _validate = _interopRequireDefault(require("uuid/dist/validate.js"));
var _version = _interopRequireDefault(require("uuid/dist/version.js"));
function _interopRequireDefault(e) { return e && e.__esModule ? e : { default: e }; }
